### PR TITLE
feat/928-Removed check and fix commands

### PR DESCRIPTION
**Pull request by BAntonellini**

### CODE REVIEW

**Review by noel — CHANGES_REQUESTED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -32,12 +32,9 @@ questionary = "^1.9.0"
 yamlloader = "^1.0.0"
 pyfiglet = "^0.8.post1"
 click = "^8.0.3"
-sqlfluff = ">=0.11.2"
-pre-commit = "^2.15.0"
 rich = "^10.4.0"
 Jinja2 = "~2.11.2"
 cookiecutter = "^1.7.3"
```
[the remark this answers is not in the document]
Comment (ssassi) [reply pyproject.toml:37]: Yes, we can remove it.

File: pyproject.toml [hunk 2]
```diff
@@ -32,12 +32,9 @@ questionary = "^1.9.0"
...
-sqlfluff-templater-dbt = ">=0.11.2"
 python-slugify = "<5.0.0"
 dbt-core = ">=0.18.0,<2.0.0"
```
[the remark this answers is not in the document]
Comment (ssassi) [reply pyproject.toml:39]: Agree

[the remark this answers is not in the document]
Comment (BAntonellini) [reply pyproject.toml:39]: @ssassi @noel `sqlfluff-templater-dbt` is one of the removed packages (alongside sqlfluff and pre-commit)

what do you mean by `doing >=1.0.0`

[the remark this answers is not in the document]
Comment (ssassi) [reply pyproject.toml:39]: dbt-core = ">=1.0.0,<2.0.0"

[the remark this answers is not in the document]
Comment (BAntonellini) [reply pyproject.toml]: Thought he referred to sqlfluff-templater-dbt

done

[the remark this answers is not in the document]
Comment (noel) [reply pyproject.toml]: the comment was against the dbt-core line :)

**Pull request merged**